Update frequently-asked-questions.md (#6375)
umbraco/UmbracoDocs: umbraco-cloud/frequently-asked-questions.md

It is strictly forbidden to attempt to do a denial of service attack on your Cloud sites.

### Can Umbraco Cloud support my website?

> The ability of Umbraco Cloud to support your website depends on several factors, including the number of visitors and the amount of media storage your site requires. However, it's important to note that these details alone (like 'X' visitors per month and 'Y' TB of media storage) don't fully determine whether Umbraco Cloud is the right fit for your needs. 

Reviewer: A suggestion to write shorter sentences.

```suggestion
The ability of Umbraco Cloud to support your website depends on different factors. This includes the number of visitors and the media storage your site requires.

These details alone do not fully determine whether Umbraco Cloud is the right fit for your needs. 
```

> Each website can have different performance requirements based on how it's built and configured. To get a better understanding of whether Umbraco Cloud can handle your specific website, we recommend running a load test. This test simulates real-world traffic and can help you understand the amount of computation power and resources your website will require. 

Reviewer: ```suggestion
Each website can have different performance requirements based on how it is built and configured. We recommend running a load test, to learn whether Umbraco Cloud can handle your specific website. This test simulates real-world traffic and can help you understand the computation power and resources your website will require. 
```

> There are user-friendly tools available, such as [LoadNinja](https://loadninja.com/), which can help you conduct these simulations. By doing this, you'll have a much clearer picture of whether Umbraco Cloud can support your website effectively.

Reviewer: ```suggestion
There are user-friendly tools available, such as [LoadNinja](https://loadninja.com/), to help you conduct these simulations. This will give you a clearer picture of whether Umbraco Cloud can support your website effectively.
```
